app/models.py

```diff
@@ -1,16 +1,52 @@
-from app import db
+from datetime import datetime
+from app import db, login
+from flask_login import UserMixin
+from werkzeug.security import generate_password_hash, check_password_hash
 
 class Task(db.Model):
+    __tablename__ = 'tasks'
+
     id = db.Column(db.Integer, primary_key=True)
-    task_description = db.Column(db.String(64), index=True)
-    due_date = db.Column(db.Date, nullable= False)
+    task_description = db.Column(db.String(256), index=True)
+    due_date = db.Column(db.DateTime, nullable= True)
     created_by = db.Column(db.String(128), nullable= False)
     status = db.Column(db.Boolean)
-    deleted_by = db.Column(db.String(128))
-    created_at = db.Column(db.Text, nullable= False)
-    deleted_at = db.Column(db.Text, nullable= True)
+    deleted_by = db.Column(db.String(128), nullable= True)
+    created_at = db.Column(db.DateTime, nullable= True)
```

Comment:
check how to set default values in models and use the default value as the current time here